assets/dev/js/editor/document/hooks/ui/settings/set-direction-mode.js

```diff
@@ -17,8 +17,14 @@ export class SetDirectionMode extends After {
 	}
 
 	apply( args ) {
+		SetDirectionMode.set( args.container?.view );
```

Comment:
The `settings` command requires a `container` so, it must exist. Also, it's better to use the container itself.

```suggestion
		SetDirectionMode.set( args.container );
```

Reply:
Done